invenio_oaiserver/config.py

```diff
@@ -41,7 +41,7 @@
     'identity',
 ]
 
-OAISERVER_RESUMPTION_TOKEN_EXPIRE_TIME = 60 * 60  # time in seconds
+OAISERVER_RESUMPTION_TOKEN_EXPIRE_TIME = 5 * 60  # time in seconds
```

Comment:
Perhaps a more aggressive 1 minute timeout be better until we know the performance impact on ES?